### ISSUE TITLE
[T&I] Verify the training meta data

### ISSUE THREAD

**Issue by Vishal-S-P** [PurdueDualityLab/tf-maskformer#20]
please verify if we have required TensorFlow code to reproduce losses reported by MaskFormer authors - 

 

- [ ] ResNet101 backbone - https://dl.fbaipublicfiles.com/maskformer/panoptic-coco/maskformer_panoptic_R101_bs64_554k/metrics.json

- [ ] ResNet50 backbone - https://dl.fbaipublicfiles.com/maskformer/panoptic-coco/maskformer_panoptic_R50_bs64_554k/metrics.json

Questions -

- [ ] What are multiple loss functions (for eg - loss_dice_1 , loss_dice_2 etc) ?
